src/sentry/static/sentry/app/views/eventsV2/index.tsx

```diff
@@ -92,6 +95,21 @@ class EventsV2 extends React.Component<Props> {
     return [t('Discover')];
   };
 
+  renderBanner() {
```

Comment:
We should move `localStorage.getItem('banner-dismissed');` here as a state e.g. `bannerHidden`. 

Then update `state.bannerHidden` to be `true` whenever `onClickDismiss` handler function is called (it'll be passed as a prop); and in addition set `localStorage.setItem('banner-dismissed', "true");`.

Reply:
Danny helped me with setting the state

Reply:
Sweet. Kudos to Danny 👍